### DISCUSSION TITLE
Remove or add bodies to ephemeridies.

### DISCUSSION THREAD

**Discussion by offwa** [skyfielders/python-skyfield#999] [Ideas]
Remove a body (and every orbitee it has or a warning telling you not to remove it), Add a body (add a body from another ephemeris)

**Comment by brandon-rhodes** [comment]
Do you mean, add or remove the body from Skyfield's in-memory list of the segments in the file? Or, do you mean, actually edit the file on-disk, the `.bsp` file, so that it gains or loses segments for various bodies?

**Reply by offwa** [reply]
I want to edit the file on-disk, since some ephermeridies have too much or too little bodies

**Reply by lulunac27a** [reply]
`.bsp` files are very large data, which it can be over 10 megabytes (10 MB) which is a lot of data and `.bsp` file is a binary file format.
